queue_services/account-mailer/README.md

```diff
@@ -0,0 +1,75 @@
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+
+# Application Name
+
+BC Registries Names Examination, research and approval system API
```

Comment:
Needs to be changed. Looks like pay also have this copy error. :)

Reply:
i will do it in next PR